app/views/general_shopping_lists/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<p style="color: green"><%= notice %></p>
+
+<div id="recipes">
+  <h1>Shopping List</h1>
+  <div class="topBoxRecipeList">
+    <div class="leftSideRecipeList">
+      <p>Amount of food items to buy: <%= @missing_foods_price %> </p>
+    </div>
+
+    <div class="rightSideRecipeList">
+      <p>Total value of food needed: <%= @missing_foods_count %> </p>
+    </div>
+  </div>
```

Comment:
- [ ] According to what's on the wireframe, it seems your calculations are not correct.
- Demo

| Yours | Wireframe 👍  |
| --- | --- |
| ![Capture](https://user-images.githubusercontent.com/63087491/219881744-2dd5b90d-ebb8-4b46-888d-7e05ce434627.PNG) | ![Capture](https://user-images.githubusercontent.com/63087491/219881809-fd8c35c9-9efd-40cb-be34-7730a238ee24.PNG) |

Reply:
Thank you for your feedback @FrederickMih... I think the left side should count the number of items, not the total quantity. And thankyou for identifying that we have swapped values in both columns. We'll surely fix that shortly.